src/components/shared/Entity/DetailsForm/Form.tsx

```diff
@@ -33,10 +33,11 @@ export const CONFIG_EDITOR_ID = 'endpointConfigEditor';
 
 export const getConnectorImageDetails = (
     connector: ConnectorWithTagDetailQuery
-) => {
+): Details['data']['connectorImage'] => {
     return {
         connectorId: connector.id,
         id: connector.connector_tags[0].id,
+        imageName: connector.image_name,
```

Comment:
Setting imageName. Eventually we should store the name and tag separate and have a computed value for imagePath but not a huge deal right now.